cmd/wio/toolchain/npm/publish/types.go

```diff
@@ -0,0 +1,42 @@
+package publish
+
+import (
+    "fmt"
+    "wio/cmd/wio/toolchain/npm"
+)
+
+type Attachment struct {
+    Type   string `json:"content-type"`
+    Data   string `json:"data"`
+    Length uint64 `json:"length"`
+}
+
+type Data struct {
+    Id          string `json:"_id"`
```

Comment:
should this be _id or id?

Reply:
it's pretty hard to accidentally insert an underscore

Reply:
true